### ISSUE TITLE
rule does not exist for boost graph

### ISSUE THREAD

**Issue by luxe** [nelhage/rules_boost#101]
I don't see a rule for "graph" anywhere in `BUILD.boost`.  
Was it left out on purpose?  

was hoping to access it with something like:  
```
  deps = ["@boost//:graph"]
```

I was able to get it to work on my own, but it was a little messy
building and pulling in other boost dependencies as separate repos
for graph specifically, it looked something like:  

**getting repo:**  
```
new_git_repository(
    name = "com_github_boost_graph",
    build_file = "bazel/deps/boost/boost_graph.BUILD",
    commit = "4c752ee2f11c249e0d5b1fab8f0ef9da3428f575",
    remote = "https://github.com/boostorg/graph.git",
)
```

**build rule:**  

```
package(default_visibility = ["//visibility:public"])

cc_library(
  name = "graph",
  includes = [
    "include/",
  ],
  hdrs = glob([
    "include/boost/**/*.hpp",
  ]),
  srcs = [
  ],
  deps = [
    "@com_github_boost_property_map//:property_map",
    "@com_github_boost_property_tree//:property_tree",
    "@com_github_boost_lexical_cast//:lexical_cast",
    "@com_github_boost_xpressive//:xpressive",
  ]
)
```

any plans to add a rule for graph?

**Comment by luxe** [comment]
I'll make a diff and see what circle ci says:   
https://github.com/nelhage/rules_boost/pull/102

**Comment by nelhage** [comment]
closed by #102
